functions/src/triggers/userCreate.ts

```diff
@@ -0,0 +1,16 @@
+import * as functions from 'firebase-functions'
+import * as jsonWebToken from 'jsonwebtoken'
+import { addTokenToUser } from '../db/addTokenToUser'
+import { assertJWTEnv } from '../utils/assertConfig'
+
+export const userCreate = functions.auth.user().onCreate(async (user) => {
```

Comment:
```suggestion
export const userCreate = functions.auth.user().onCreate(async (user: JWTData) => {
```

Reply:
if this function is a hook, I would say `onUserCreate`